openadapt/app/cards.py

```diff
@@ -7,7 +12,12 @@
 PROC = None
 
 
-def settings(dark_mode):
+def settings(dark_mode: bool) -> None:
+    """Display the settings dialog.
+
+    Args:
+        dark_mode (bool): Current dark mode setting.
+    """
     with ui.dialog() as settings, ui.card():
```

Comment:
Should there be a newline here? I would have thought `black` would catch this.

(I will not mention this again elsewhere in this PR.)

Reply:
@abrichr I am following [pydocstyle(flake8-doctstrings) ](http://www.pydocstyle.org/en/2.1.1/error_codes.html)that says
> D202: No blank lines allowed after function docstring
If you'd like I can ignore this error and add newlines to subsequent function docstrings. There is no mention of newlines to multidocstrings in PEP, only that there should be no newlines for single line docstrings.